unity-renderer/Assets/Scripts/MainScripts/DCL/Bridges/DataStoreBridge/DataStoreBridge.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using DCL;
+using UnityEngine;
+
+public class DataStoreBridge : MonoBehaviour
```

Comment:
Not sure about this name. Shouldn't this be something related to `BuilderInWorld`  rather than the broad concept of `DataStore`?

Reply:
Truth be told. I didn't wanted to add a MonoBehaviour just for this.
I was thinking... How many times did I had to create a MonoBeaviour just to set a variable from the DataStore with data coming from kernel...
So... Why not creating a script where we can do just that?
That's the main reason for this. What do you think? Is it a good idea or should I change it?